packages/genesys-spark-components/src/components/beta/gux-side-panel/example.html

```diff
@@ -0,0 +1,143 @@
+<div id="side-panel-wrapper">
+  <div id="content">
+    <h1>Side Panel</h1>
+
+    <p>
+      You can listen to the 'sidePanelDismiss' event on the
+      `gux-side-panel-beta` element to know when the side panel is to be
+      hidden/removed.
+    </p>
+
+    <gux-cta-group>
+      <gux-button slot="secondary" id="toggle-small">Toggle small</gux-button>
+      <gux-button slot="secondary" id="toggle-medium">Toggle medium</gux-button>
+    </gux-cta-group>
+
+    <h2>Modal Side Panel</h2>
+
+    <p>You can show/hide the `gux-modal-side-panel` via two mechanisms:</p>
+    <ol>
+      <li>
+        Use the `showModal()` and `close()` methods on a `gux-modal-side-panel`
+        element
+      </li>
+      <li>Set the `open` property or attribute on `gux-modal-side-panel`</li>
+    </ol>
+
+    <gux-modal-side-panel-beta id="modal-example">
+      <gux-side-panel-heading
+        slot="heading"
+        level="3"
+        icon="fa/diamond-regular"
+      >
+        Side panel title
+      </gux-side-panel-heading>
+      <h2 slot="description">Pellentesque habitant morbi tristique sen</h2>
```

Comment:
I was thinking since you have `gux-side-panel-heading` would it be good to also have `gux-side-panel-description` that just has a default slot. I know users will be able to slot just text nodes but there is ways around it like it had been done in the `gux-form` component. I think what you have is fine just said I would mention it.

Reply:
@gavin-everett-genesys Thanks for the suggestion. I don't think its worth it, the description is really just a way for UX to easily add text, I would expect devs to just use the slot.